cypher/cypher-docs/src/docs/dev/ql/administration/security/role-management-syntax.asciidoc

```diff
@@ -6,8 +6,9 @@
 | [source, cypher]
 ----
 SHOW [ALL\|POPULATED] ROLES
-    [YIELD field1[, field2] [ORDER BY field1 [, field2]] [SKIP n] [LIMIT n]]
+    [YIELD { * \| field1[, field2] } [ORDER BY field1 [, field2]] [SKIP n] [LIMIT n]]
```

Comment:
In the security documentation, we do `[, ...]` instead of ` [, field2]` to mark that it could be one or several values. I don't have a super strong opinion on this but if you read it literally I guess this way of writing it would suggest that you can have either 1 or 2 values but not 3.